nanshe_ipython.ipynb

```diff
@@ -367,6 +368,81 @@
       "cell_type": "markdown",
       "metadata": {},
       "source": [
+        "### Trimming\n",
+        "\n",
+        "* `front` (`int`): amount to trim off the front\n",
+        "* `back` (`int`): amount to trim off the back\n",
+        "\n",
+        "<br>\n",
+        "* `block_frames` (`int`): number of frames to work with in each block (run in parallel).\n",
+        "* `norm_frames` (`int`): number of frames for use during normalization of each full frame block (run in parallel)."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "%%time\n",
+        "\n",
+        "\n",
+        "front = 0\n",
+        "back = 0\n",
+        "\n",
+        "block_frames = 1\n",
+        "norm_frames = 100\n",
+        "\n",
+        "\n",
+        "# Somehow we can't overwrite the file in the container so this is needed.\n",
+        "io_remove(data_basename + postfix_trim + zarr_ext)\n",
+        "io_remove(data_basename + postfix_trim + h5_ext)\n",
+        "\n",
+        "\n",
+        "with open_zarr(data_basename + zarr_ext, \"r\") as f:\n",
+        "    with get_executor(client) as executor:\n",
+        "        # Load and prep data for computation.\n",
+        "        imgs = f[\"images\"]\n",
```

Comment:
Should have been `dataset`. Fixing in PR ( https://github.com/nanshe-org/nanshe_workflow/pull/153 ).